examples/managed-identity/kubernetes.json

```diff
@@ -0,0 +1,38 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "kubernetesConfig": {
+        "useManagedIdentity": true

```

Comment:
Let me think on it and chat with you more about it. I'd be fine with either of those approaches and don't have any strong opinions here.